tox.ini

```diff
@@ -1,12 +1,13 @@
 [tox]
-envlist = py{27,33,34,35,36,37},codechecks
+envlist = py{35,36,37},codechecks
 
 [testenv]
 deps=
     pytest
     pytest-cov
     coveralls
     mock
+    parsedatetime
```

Comment:
I don't like doubly defining this, trying to figure out why tox isn't pulling it from setup.py - even when `python setup.py develop` is explicitly called in tox.